### DISCUSSION TITLE
Show Disclaimer on Login Screen

### DISCUSSION THREAD

**Discussion by kompetenzlandkarte** [dani-garcia/vaultwarden#1893] [General]
Hello together,

we have a vaultwarden instance with 100+ Users. We are looking into the possibility to show a disclaimer on the login screen which shows a generic message in an alert format like:

```
If you need assistance. Please message foo@bar.com.
```

Is this possible somehow?

Thank you.

**Comment by jjlin** [comment]
You would have to modify the web vault source and build your own version of it. You probably want to modify `src/app/layouts/frontend-layout.component.html`. See https://github.com/dani-garcia/vaultwarden/wiki/Building-binary#install-the-web-vault for build instructions.

**Comment by pepa65** [comment]
What might wordk for you is to modify the local `/web-vault/app/main.*.js.map` in Docker on startup and insert your snippet after `<a routerLink=\"/hint\">{{'getMasterPasswordHint' | i18n}}</a>\n`.
